sdk/storage/inc/datalake/datalake_options.hpp

```diff
@@ -173,12 +184,12 @@ namespace Azure { namespace Storage { namespace DataLake {
     /**
      * @brief Specify the transactional md5 for the body, to be validated by the service.
      */
-    std::string ContentMD5;
+    Azure::Core::Nullable<std::string> ContentMD5;
```

Comment:
Can you double-check CRC64 is not supported?

Reply:
https://docs.microsoft.com/en-us/rest/api/storageservices/datalakestoragegen2/path/update
Currently not supported.